Code/EnginePlugins/PhysXPlugin/WorldModule/Implementation/PhysXWorldModule.cpp

```diff
@@ -1085,6 +738,16 @@ void ezPhysXWorldModule::StartSimulation(const ezWorldModule::UpdateContext& con
 
 void ezPhysXWorldModule::FetchResults(const ezWorldModule::UpdateContext& context)
 {
+  EZ_PROFILE_SCOPE("FetchResults");
+
+  if (const ezCameraComponentManager* pCompMan = GetWorld()->GetComponentManager<ezCameraComponentManager>())

```

Comment:
Use ezRenderWorld::GetViewByUsageHint with mainview and editor view and the world instead. Then get the camera and position from it. There is no guarantee that their is a camera component for the main view. And above will also work in simulation mode in the editor.

Reply:
Works indeed better.